elfloader-tool/src/arch-arm/main.c

```diff
@@ -148,32 +204,8 @@ static paddr_t load_elf(const char *name, void *elf,
     info->virt_entry = (vaddr_t)elf_getEntryPoint(elf);
     info->phys_virt_offset = dest_paddr - (vaddr_t)min_vaddr;
 
-    /* Round up the destination address to the next page */
```

Comment:
We started this work a while ago on an older commit. It may have been removed when rebasing the IBLL and Secure Load. Putting it back.